### PR TITLE
[Bromley] Update assets JS to handle new sub-categories

### PR DESCRIPTION
**Pull request by chrismytton**
Bromley are switching to using "normal" FixMyStreet subcategories, which means that whereas previously they had categories and the subcategory was an extra question, now the categories have groups, so they behave like regular FMS category/subcategory.

This change updates the assets JS to use the new category names and updates the JS tests to reflect the new way Bromley is setup on staging.

**Note**: This should go live when Bromley are ready to switch over their Open311 endpoint, preferably at the same time as the switchover in the admin.

Part of https://github.com/mysociety/societyworks/issues/2706

[skip changelog]

### CODE REVIEW

**Review by dracos — APPROVED** [review]
One test refactor suggestion, but no need to be done now if no time :)

File: .cypress/cypress/integration/bromley.js [hunk 1]
```diff
@@ -15,13 +15,15 @@ describe('Bromley cobrand', function() {
...
+    cy.nextPageReporting();
+    cy.get('#subcategory_StreetLightingandRoadSigns label').contains('Lamp Column Damaged').click();
     cy.get('#form_prow_reference').should('have.value', 'FP111');
   });
 
   it('does not display asset based upon extra question', function() {
     cy.pickCategory('Street Lighting and Road Signs');
     cy.nextPageReporting();
-    cy.pickSubcategory('Non-asset', '#form_service_sub_code');
+    cy.get('#subcategory_StreetLightingandRoadSigns label').contains('Sign Light Not Working').click();
```
[the remark this answers is not in the document]
Comment (chrismytton) [reply .cypress/cypress/integration/bromley.js]: Yeah definitely worth factoring out at this point. Have done so in 0a2a408c9a30ddc0f56f6e6c0fdef5b6f5c35d35

**Comment by dracos** [comment]
@davea Could you review the last commit of this, https://github.com/mysociety/fixmystreet/pull/3832/commits/2fa5357d2af747e9046ec600bf7c8b53dd436fc7, for the rest of https://github.com/mysociety/societyworks/issues/2706 - hopefully removes all the Bromley subcategory code, and moves the "default to user's categories" code to core Dashboard code (I asked a qn on Slack as to whether we should move this to core Reports code so it applied to All reports page or not, but that can be in future). Thanks :)

**Review by davea — APPROVED** [review]
🐱 🆗

**Comment by davea** [comment]
Thinking about it more, the "All Reports" link does already take the user to a pre-filtered map thanks to the `inspector_redirect` thing. No reason that couldn't be replaced in future with a simple link to `/reports` that does the filtering there.

**Pull request merged**